warehouse/warehouse.go

```diff
@@ -667,60 +666,6 @@ func (wh *HandleT) createJobs(warehouse warehouseutils.WarehouseT) (err error) {
 	return nil
 }
 
-func (wh *HandleT) sortWarehousesByOldestUnSyncedEventAt() (err error) {
```

Comment:
This seems to be a useful utility to find uploads with oldest event. Incase we want to build fairness checks in future, this might be a good start. Is there any value in exposing this through rudder-cli? If not, move it to our debug repo?